### PR TITLE
comment out ccd int referencs

### PR DESCRIPTION
**Pull request by NickAzureDevops**
### Jira link

See [DTSPO-18669](https://tools.hmcts.net/jira/browse/DTSPO-18669)

### Change description

Remove ccd int references as no it is longer required. will fix issue with hr as part of upgrading aks ticket 

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

## 🤖AEP PR SUMMARY🤖

_I'm a bot that generates AI summaries of pull requests, see [AEP](https://kainossoftwareltd.github.io/ai-enhanced-platform/) for more details_

### apps/ccd/automation/kustomization.yaml
- Removed `./ccd-ac-int-data-store-api/image-repo.yaml` and `./ccd-ac-int-data-store-api/demo-image-policy.yaml`.
- Added `./ccd-test-stubs-service/image-repo.yaml` and `./ccd-test-stubs-service/image-policy.yaml`.
- Updated references for `./ccd-definition-store-api/demo-image-policy.yaml`, `./ccd-definition-store-api/image-policy.yaml`, and `./ccd-definition-store-api/ithc-image-policy.yaml`.
- Removed `./ccd-ac-int-api-gateway-web/image-repo.yaml` and `./ccd-ac-int-api-gateway-web/image-policy.yaml`.
- Added `./message-publisher/image-repo.yaml`, `./message-publisher/image-policy.yaml`, and `./message-publisher/demo-image-policy.yaml`.

### CODE REVIEW

**Review by thomast1906 — CHANGES_REQUESTED** [review]
content added from your local?

File: apps/ccd/automation/kustomization.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../ccd-ac-int-data-store-api/image-repo.yaml
-  - ../ccd-ac-int-data-store-api/demo-image-policy.yaml
+  # - ../ccd-ac-int-data-store-api/image-repo.yaml
+  # - ../ccd-ac-int-data-store-api/demo-image-policy.yaml
```
Comment (madjava) [inline apps/ccd/automation/kustomization.yaml:5]: Hey @NickAzureDevops i see commented out code `#` is this as expected? comments says uncommented out code

File: apps/ccd/automation/kustomization.yaml [hunk 2]
```diff
@@ -16,8 +16,8 @@ resources:
   - ../ccd-definition-store-api/demo-image-policy.yaml
   - ../ccd-definition-store-api/image-policy.yaml
   - ../ccd-definition-store-api/ithc-image-policy.yaml
-  - ../ccd-ac-int-api-gateway-web/image-repo.yaml
-  - ../ccd-ac-int-api-gateway-web/image-policy.yaml
+  # - ../ccd-ac-int-api-gateway-web/image-repo.yaml
+  # - ../ccd-ac-int-api-gateway-web/image-policy.yaml
```
Comment (madjava) [inline apps/ccd/automation/kustomization.yaml:20]: @NickAzureDevops same as above, there are still comments, as expected?

File: apps/ccd/automation/kustomization.yaml (hunk 1 above)
Comment (NickAzureDevops) [reply apps/ccd/automation/kustomization.yaml:5]: meant to be commented out and not the other way around my bad

File: apps/ccd/automation/kustomization.yaml (hunk 2 above)
Comment (NickAzureDevops) [reply apps/ccd/automation/kustomization.yaml:20]: > @NickAzureDevops same as above, there are still comments, as expected?

i did it the same like this https://github.com/hmcts/cnp-flux-config/blob/master/apps/ccd/automation/kustomization.yaml as one got commented out too @madjava

File: apps/ccd/automation/kustomization.yaml (hunk 1 above)
Comment (madjava) [reply apps/ccd/automation/kustomization.yaml:5]: Oh i see no longer needed so removing

**Pull request merged**